### ISSUE TITLE
Images are not loaded when using a CSS grid layout

### ISSUE THREAD

**Issue by markhowellsmead** [tuupola/jquery_lazyload#251]
The images at the top of e.g. the third column only appear when the visitor scrolls a little way down the page: presumably, to where the image would be if CSS `column-count` were not active.

**Comment by tuupola** [comment]
URL of the page you have problem with? Have you tried increasing value `failure_limit`parameter? Check *"When images are not sequential"* in the [documentation](http://www.appelsiini.net/projects/lazyload).

**Comment by mhmli** [comment]
http://www.mhm.li/dev/pt10/

Options for `skip_hidden` and `failure_limitparameter` are set per http://www.mhm.li/dev/pt10/js/ui.js.

**Comment by tuupola** [comment]
Page does not show any photos at the moment?

**Comment by mhmli** [comment]
Oops. I've re-initialized them.

**Comment by mhmli** [comment]
My bad. These parameters are now corrected.

How can I set a width and height when the whole layout is flexible and dependent upon the browser size?

**Comment by mciszczon** [comment]
I have set the dimensions to percentages (`width: 100%; height: auto;`) and it works, aaaand is responsive (or call it *flexible*). It's not said anywhere that these should be exact values.

I had the same problem with my site. I've got a nice grid layout with photos, created using **Salvattore** plugin. But **Lazy Load** only loaded images in the first column, the rest only appeared after scrolling a little bit down.

Adding `failure_limit : 10` solved the problem for me.
